core/src/test/java/org/mskcc/cbio/portal/scripts/TestIntegrationTest.java

```diff
@@ -180,7 +180,7 @@ public void testLoadStudyEs0() throws Throwable {
             geneticProfileStableIds = Collections.nCopies(sampleIds.size(), "study_es_0_structural_variants");
 
             StructuralVariantService structuralVariantService = applicationContext.getBean(StructuralVariantService.class);
-            List<StructuralVariant> structuralVariants = structuralVariantService.fetchStructuralVariants(geneticProfileStableIds, sampleIds, entrezGeneIds);
+            List<StructuralVariant> structuralVariants = structuralVariantService.fetchStructuralVariants(geneticProfileStableIds, sampleIds, entrezGeneIds, new ArrayList<>());
```

Comment:
Perhaps give this a name (move to variable)?

Reply:
Fixed in [753dfd6](https://github.com/cBioPortal/cbioportal/commit/753dfd694e07b700efddaa51163609e8de89ff11)